docs/reuse/apps/app-install-v2.md

```diff
@@ -0,0 +1,16 @@
+To install the app, do the following:
+
+1. From the Sumo Logic navigation, select **App Catalog**.
+1. In the **Search Apps** field, search for and then select your app.
+1. Optionally, you can scroll down to preview the dashboards included with the app. 
+2. To install the app, click on **Install App**. 
+3. Click on **Next** in **Setup Data** section.
+4. In the **Configure** section of respective app, complete the following fields.
+    1. **Key.** Select either of these options for the data source.
+        * Choose **Source Category**, and select a source category from the list for **Default Value**.
+        * Choose **Custom**, and enter a custom metadata field beginning with an underscore. Insert its value in **Default Value**.
```

Comment:
remove "beginning with an underscore" since it can be any metadata for example db.cluster, container.name etc we can add these as example